_includes/data-privacy.html

```diff
@@ -0,0 +1,85 @@
+<section id="data-privacy">
+	<div class="container">
+		<div class="row">
+			<div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
+				<h2 class="section-heading text-center">Datenschutzerklärung</h2>
+				<hr class="primary">
+				<div class="row">
+					<div class="col-md-12">
+						<p>
+							Diese Datenschutzerklärung soll Sie als Nutzer dieser Website über die Art, den Umfang und den
+							Zweck der Erhebung und Verwendung personenbezogener Daten durch den Websitebetreiber Kinderladen
+							Fuchsbau informieren. Der Websitebetreiber nimmt Ihren Datenschutz sehr ernst und behandelt Ihre
+							personenbezogenen Daten vertraulich und entsprechend der gesetzlichen Vorschriften.
+						</p>
+
+						<h3>Personenbezogene Daten</h3>
+						<p>
+							Diese Website können Sie besuchen, ohne Angaben zu Ihrer Person zu machen. Es werden jedoch Ihre
+							Zugriffsdaten auf diese Website gespeichert (ohne Personenbezug). Zu diesen Zugriffsdaten gehören
+							z. B. die von Ihnen angeforderte Datei oder der Name Ihres Internet-Providers.
+
+							Verarbeitung personenbezogener Daten
```

Comment:
Ich glaube das hier sollte ein `<h3>` oder so. Für die Lesbarkeit wäre bei solch langen Texten vielleicht das Markdown plugin für jekyll gut, dann könnte das alles in Markdown geschrieben werden. Jedenfalls ist hier ein kleiner Bug, siehe auch Screenshot.
![selection_002](https://user-images.githubusercontent.com/1150767/40572282-3f3a6114-60a8-11e8-852e-de2e978005d5.jpg)